[feat] Add logo to documentation

kunzaatko opened the pull request:
I added a logo that I generated to with ChatGPT to the documentation. Feel free to close without merging if this is not something that you would like :)
This is the resulting logo:
![logo](https://github.com/user-attachments/assets/975031b2-3173-4f4d-8816-0c857908f345)

elsoroka commented:
hmm, interesting. I've never thought about this package needing a logo, but it's a good idea. Commenting so I remember to think about where a logo should go.

mykelk commented:
What if we use the Julia shades of red, green, and purple to emphasize the tie to the Julia programming language?

kunzaatko commented:
I changed the font a bit and made it look a bit more modern...
I adjusted the colours to match the julia shades exactly. Here is for purple:
![logo_purple](https://github.com/user-attachments/assets/3a4016ac-41eb-4a06-a432-9b5f81eafc3d)
and here for blue:
![logo_blue](https://github.com/user-attachments/assets/5dafcd84-7a25-4605-9d86-7a4f8fa783c5)
Which one do you prefer? @mykelk

kunzaatko commented:
> I wonder if it would make sense to use the Julia traditional three-dot form. The top dot could have a checkmark in it, and the bottom two dots could be labeled with T and F.
In my opinion this would be counter productive as it would necessarily make the letters and the checkmark smaller hence less recognizable as such... But if you both agree on it, it is not too hard to change it so.

kunzaatko commented:
What do you think about this @elsoroka ? Do you want the logo or should we close this as not planned?

kunzaatko commented:
I will also add a PR for the documentation `Project.toml` to resolve it from the relative path. It is better since it avoids mismatch of the code of the documentation with the package code.

elsoroka commented:
Hmm, I merged but I don't see the logo. I was hoping it would come up on the navigation menu in the docs. I will look into why this happens...
...Also if anyone, anywhere, knows how to teach Apple Mail that even though I delete Github notifications from my mailbox, I actually want to see them and not have them automatically junked, please let me know. that's how pull requests get missed.

kunzaatko commented:
The documentation was still building. It should be visible now
